Use more specific markup for ins/del related to amendments

frivoal opened the pull request:
Instead of the idiosyncratic use of the class attribute to track which ins/del belong to which amendment, use the cite attribute, which is made for that:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ins#attr-cite
https://html.spec.whatwg.org/#attr-mod-cite

This change can be made without regards for compatibility as this style of markup has not yet been deployed.

frivoal committed:
Use more specific markup for ins/del related to amendments

Instead of the idiosyncratic use of the class attribute to track which
ins/del belong to which amendment, use the cite attribute, which is made
for that:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ins#attr-cite
https://html.spec.whatwg.org/#attr-mod-cite

This change can be made without regards for compatibility as this style of
markup has not yet been deployed.

fantasai commented:
By making this switch, actually, we can drop the requirement to start the ID with a 'c'.

frivoal committed:
No need to namespace the amendments by starting them with a "c" anymore

This was useful if we had to use the class attribute, as all sorts of
things go there, but now that we use the cite attribute, that takes care
of the potential collisions, and there's no need to mandate starting with
a "c" anymore.
